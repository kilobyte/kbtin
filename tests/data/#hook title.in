#hook close {#showme FIN;#end}
#hook title {#showme {TITLE: %1}}
#rem [[[ (m4 quirk)
#run a {sh -c 'printf "\033]0;good beer is good\033\\\\\033]1;junk\033\\\\\033]0;meow\a"'}
