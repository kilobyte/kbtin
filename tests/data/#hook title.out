TITLE: good beer is good
TITLE: meow
FIN
